modules/ding_wayf_dk/includes/DingWayfGateway.class.inc

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * @file
+ * Simple class for communication with the SimpleSAMLphp DBC gateway.
```

Comment:
If this class deals with the SimpleSAMLphp DBC gateway then why is it included in this module which uses wayf.dk?